packages/ui/src/components/Overlay.vue

```diff
@@ -9,7 +9,7 @@ defineProps<OverlayProps>()
 
 <template>
   <div
-    class="fixed left-0 top-0 $ui-z-max h-full w-full $ui-fcc transition-opacity transition-duration-300 $ui-bg-base"
+    class="absolute left-0 top-0 $ui-z-max h-full w-full $ui-fcc transition-opacity transition-duration-300 $ui-bg-base"
```

Comment:
Why did we change the position type, I think keeping it as fixed position type will be more flexible and easier to use.

Reply:
fixed has an issue，see the picture below

![image](https://github.com/vuejs/devtools-next/assets/49969959/ae9c979d-69dd-4c24-9063-0306752260c3)

the top or left will be calc by viewport，that will cause position error on split screen.

Maybe we can make absolute as an option？

Reply:
Sure, we can add an option to handle it.

Reply:
Added